src/main/src/main/java/com/shevtsov/dao/impl/DBConnectionWorkDB.java

```diff
@@ -0,0 +1,19 @@
+package com.shevtsov.dao.impl;
+
+import com.shevtsov.dao.DBConnection;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+
+public class DBConnectionWorkDB implements DBConnection {
+    private static final String DB_URL = "jdbc:h2:tcp://localhost/~/LuxoftShop";
```

Comment:
better read this properties from file, so you can reuse this class in tests

Reply:
do you mean I have to use local DB 'LuxoftShop' for testing? Artem said to use in-memory DB.

Reply:
No. I propose to you just retrieve db settings from file and use different files for test and prod